README.md

```diff
@@ -105,6 +105,10 @@ A Vulkan implementation is found by CMake in the following priority order:
      Example: `VULKAN_SDK=$HOME/vulkan-macos-1.1.85.0/macOS`
    * `VK_ICD_FILENAMES` should point to the ICD JSON file.
      Example: `VK_ICD_FILENAMES=$VULKAN_SDK/etc/vulkan/icd/MoltenVK_icd.json`
+   * `VK_LAYER_PATH` should point to the etc/explicit\_layer.d folder in your
```

Comment:
"`VK_LAYER_PATH` should point to `$VULKAN_SDK/etc/explicit_layer.d`"
would be more clear for this?

Reply:
done